lms/static/js/my_courses_dropdown.js

```diff
@@ -1,12 +1,23 @@
 $(document).ready(function() {
     'use strict';
 
-  // define variables for code legibility
+    // define variables for code legibility
     var dropdownMenuToggle = $('.dropdown');
     var dropdownMenu = $('.dropdown-menu');
     var menuItems = dropdownMenu.find('a');
 
-  // bind menu toggle click for later use
+    var keyCodes = {
```

Comment:
I believe you can use [`edx.constants`](http://ui-toolkit.edx.org/utilities/constants/) here. We have a window object that has a bunch of pre-defined keys.